cl/search/models.py

```diff
@@ -3338,6 +3347,11 @@ def get_absolute_url(self) -> str:
     def clean(self) -> None:
         if self.type == "":
             raise ValidationError("'type' is a required field.")
+        if self.ordering_key is not None and self.ordering_key != "":
```

Comment:
How about:

```suggestion
        if self.ordering_key:
```

Or:

```suggestion
        if isinstance(self.ordering_key, int):
```

Or, I guess, even better would be to combine these with an `and`:

```suggestion
        if isinstance(self.ordering_key, int) and self.ordering_key < 1:
            ...
```